src/jit/ee_il_dll.cpp

```diff
@@ -163,6 +163,20 @@ CorJitResult CILJit::compileMethod (
         return g_realJitCompiler->compileMethod(compHnd, methodInfo, flags, entryAddress, nativeSizeOfCode);
     }
 
+    if (!JitConfig.isInitialized())
```

Comment:
It seems like the "correct" place to put this would be as a new argument to getJit() and/or sxsJitStartup(), or to create a new "jitStartup()" api that would pass some ICorJitOneTimeStartup* interface, but I guess that would be more intrusive.